build-system/server/app.js

```diff
@@ -1479,71 +1475,6 @@ app.get('/dist/amp-inabox-host.(m?js)', (req, _res, next) => {
 /*
  * Start Cache SW LOCALDEV section
  */
```

Comment:
Can someone confirm that the handler for `/dist/diversions` can also be removed? It's the remaining handler within this block delimited by comments.

Reply:
are those routes shared or used with testing?

Reply:
cc @rsimha

Reply:
I don't see this being used by any tests. If CI is green after this, that's good enough for me 😃